### ISSUE TITLE
[req]: Aviability to add a text to entry's name

### ISSUE THREAD

**Issue by NoAdO** [Jackett/Jackett#12710]
### Is there already a request for your feature?

- [X] I have checked older issues, open and closed

### Is your feature request related to a problem? Please describe.

The problem is russian torrent trackers do not follow torrent standards. It can be understood - there's no need to write "RUS" or "Russian" sound to name if 99% names are in Russian. But when Radarr tries to parse smth like `Южный Парк: После COVID’а / South Park: Post COVID (2021)` it has no language mentioned and parsed as original-language by Radarr and not grabbed cause i ask Russian but get English. I guess the same problem can be found in any non-English tracker which is oriented to national audio.

Firstly i've asked Radarr for help but they've answered that is not a Radarr issue: Torrent was created with no sound info -> Tracker does not add it in RSS -> Jacket doesn't add any info -> there's no info. For now the only way to fix the problem in such Russian trackers is to choose "Any" language in Radarr so English will be grabbed too. 

The exact quote of solution they offer:
> Just tell them they need to add RUS or RUSSIAN to the release names for that indexer

### Describe the solution you'd like

The feature request is to add the text field for indexer and glue a text from there to the torrent name everywhere / in the feed so that Radarr could get search results like this:
`RUS Южный Парк: После COVID’а / South Park: Post COVID (2021) BDrip`
`RUS Южный Парк: После COVID’а / South Park: Post COVID (2021) 720p`
`RUS Южный Парк: После COVID’а / South Park: Post COVID (2021) CamRip медвед`
if the field has a "RUS " input.

### Describe alternatives you've considered

As an alternative i can see that Jackett does already have a tags system. Optional glueing the tags to torrent name colud be an alternative sollution.

Another way is to tune the exact russian indexers to hardcode adding "RUS" to their entry names.

**Comment by ilike2burnthing** [comment]
In a general sense - https://github.com/Jackett/Jackett/issues/11014

If there is a specific indexer this is an issue for, let us know and we can potentially add a fix for it.

**Comment by NoAdO** [comment]
Both solutions will work. I thought about 11014 as a main solution and a indexer-specific case as an alternative one.  So, yes
> If i understand correctly you want the parser to ADD or REPLACE VFQ by FRENCH like in this example :

My solution was exact this one - i want parser to add "RUSSIAN" to indexer results of name:
before: `Южный Парк: После COVID’а / South Park: Post COVID (2021) BDrip`
after: `RUSSIAN Южный Парк: После COVID’а / South Park: Post COVID (2021) BDrip`

Hope this will work for Radarr. So for main solution this case is a duplicate of 11014. 

As for indexer-specific, i need to investigate futher but if i take a Space Jam 2 for example, nnmclub indexer and rutor indexer usually return name like Космический джем: `Новое поколение / Space Jam: A New Legacy (2021) UHD BDRemux 2160p от селезень | 4K | HDR | D` which has no language specified. So Radarr false mark them as original -> English . Force adding "RUS"  . It's the common situation not to write "russian" at russian tracker because most movies are in russian. So marking ALL of them as russian will decrease lies, but not remove them at all. We will get not false English in most cases, but false Russian in some cases. That's not 100% solution. Well, neither 11014 is, but it's much more greater than nothing.

**Comment by ilike2burnthing** [comment]
[v0.20.178](https://github.com/Jackett/Jackett/releases/tag/v0.20.178)
